DARTS/perform_pred.py

```diff
@@ -1,57 +1,48 @@
+#!/usr/bin/env python
+
 import torch
 import torch.nn as nn
 import numpy as np
 import os
+from os.path import abspath, join as pjoin
 import nibabel
-from models.dense_unet_model import Single_level_densenet, Down_sample, Upsample_n_Concat, Dense_Unet
-# from models.dense_unet_model import *
-from utils import load_data, create_one_hot_seg, back_to_original_4_pred, back_to_original_4_prob, pickling
-from models.unet import Unet,Downsample_block,Upsample_block
+from DARTS.models.dense_unet_model import Single_level_densenet, Down_sample, Upsample_n_Concat, Dense_Unet
+from DARTS.utils import load_data, create_one_hot_seg, back_to_original_4_pred, back_to_original_4_prob, pickling
+from DARTS.models.unet import Unet,Downsample_block,Upsample_block
 
 import argparse
 
+def main():
 
-def str2bool(v):
-    if isinstance(v, bool):
-        return v
-    if v.lower() in ('yes', 'true', 't', 'y', '1'):
-        return True
-    elif v.lower() in ('no', 'false', 'f', 'n', '0'):
-        return False
-    else:
-        raise argparse.ArgumentTypeError('Boolean value expected.')
-
-parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser()
 
-parser.add_argument('--input_image_path', type=str,
-                    help='Path to input image (can be of .mgz or .nii.gz format)(required)')
+    parser.add_argument('--input_image_path', type=str,
+                        help='Path to input image (can be of .mgz or .nii.gz format)(required)')
 
-parser.add_argument('--segmentation_dir_path', type=str,
-                    help='Directory path to save the output segmentation (required)')
+    parser.add_argument('--segmentation_dir_path', type=str,
+                        help='Directory path to save the output segmentation (required)')
 
-parser.add_argument('--file_name', type = str,
-                    help= 'Name of the segmentation file (required)')
+    parser.add_argument('--file_name', type = str,
+                        help= 'Name of the segmentation file (required)')
 
-parser.add_argument('--model_type', type = str,default = "dense-unet",
-                    help = 'Model types: "dense-unet", "unet" (default: "dense-unet")')
+    parser.add_argument('--model_type', type = str,default = "dense-unet",
+                        help = 'Model types: "dense-unet", "unet" (default: "dense-unet")')
 
-parser.add_argument('--model_wts_path', type=str, default='./saved_model_wts/dense_unet_back2front_finetuned.pth',
-                    help="Path for model wts to be used (default='./saved_model_wts/dense_unet_back2front_finetuned.pth')")
+    parser.add_argument('--model_wts_path', type=str,
```

Comment:
I suggest removing the default entirely since you don't expect a user to clone this repo and then install.
When the user does `pip install DARTSeg`, this file will go into 
`~/miniconda3/bin` and `~/miniconda3/lib/python3*/site-packages/DARTS` directories. We shouldn't expect the user to download `saved_model_wts` into the above two. Hence, a default based on `./` is irrelevant.